packages/lesswrong/components/posts/PostsItemMeta.jsx

```diff
@@ -73,7 +73,7 @@ const PostsItemMeta = ({classes, currentUser, post, read}) => {
         <PostsStats post={post} />
       }
 
-      { afBaseScore && <MetaInfo>
+      { (afBaseScore || afBaseScore === 0) && <MetaInfo>
```

Comment:
IIUC this will cause all posts to display AF karma, even if they have no AF karma and aren't AF posts? That doesn't seem right.

Reply:
Nope, the only difference is that now we display AF karma on AF posts that have 0 karma. Previously, when an AF-post had zero karma, we wouldn't display its af karma. `afBaseScore` should be `null` for non-af posts.

Reply:
Though to be save, I can add a chec for `posts.af` there.